traffic_portal/app/src/common/service/utils/DeliveryServiceUtils.js

```diff
@@ -17,45 +17,61 @@
  * under the License.
  */
 
-var DeliveryServiceUtils = function($window, propertiesModel) {
-
-	this.protocols = {
-		0: "HTTP",
-		1: "HTTPS",
-		2: "HTTP AND HTTPS",
-		3: "HTTP TO HTTPS"
-	};
-
-	this.qstrings = {
-		0: "USE",
-		1: "IGNORE",
-		2: "DROP"
-	};
-
-	this.geoProviders = {
-		0: "MaxMind",
-		1: "Neustar"
-	};
-
-	this.geoLimits = {
-		0: "None",
-		1: "CZF Only",
-		2: "CZF + Country Code(s)",
-	};
-
-	this.rrhs = {
-		0: "no cache",
-		1: "background_fetch",
-		2: "cache_range_requests",
-		3: "slice"
-	};
-
-	this.openCharts = function(ds) {
-		$window.open(
-			propertiesModel.properties.deliveryServices.charts.customLink.baseUrl + ds.xmlId,
-			'_blank'
-		);
-	};
+var DeliveryServiceUtils = function ($window, propertiesModel) {
+
+    this.protocols = {
+        0: "HTTP",
+        1: "HTTPS",
+        2: "HTTP AND HTTPS",
+        3: "HTTP TO HTTPS"
+    };
+
+    this.qstrings = {
+        0: "USE",
+        1: "IGNORE",
+        2: "DROP"
+    };
+
+    this.geoProviders = {
+        0: "MaxMind",
+        1: "Neustar"
+    };
+
+    this.geoLimits = {
+        0: "None",
+        1: "CZF Only",
+        2: "CZF + Country Code(s)",
+    };
+
+    this.rrhs = {
+        0: "no cache",
+        1: "background_fetch",
+        2: "cache_range_requests",
+        3: "slice"
```

Comment:
these whitespace changes are as evil as they are unnecessary. Just use tabs when adding to a file that uses real indentation.

Reply:
Thought I had my editor set to tabs for js so I just assumed it converted them to tabs.

Reply:
now THAT I would've gotten behind